<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
    <item>
     <widget class="Header" name="w_header" native="true"/>
    </item>
    <item>
     <widget class="QStackedWidget" name="sw_pages">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="LoginForm" name="page_login"/>
      <widget class="TotalTime" name="page_totalTime"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TotalTime</class>
   <extends>QWidget</extends>
   <header>totaltime.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LoginForm</class>
   <extends>QWidget</extends>
   <header>loginform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Header</class>
   <extends>QWidget</extends>
   <header>header.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
